returnn/datasets/postprocessing.py

```diff
@@ -237,9 +245,12 @@ def _iterate_dataset(self) -> Iterator[TensorDict]:
         seq_index = 0
         while self._dataset.is_less_than_num_seqs(seq_index):
             self._dataset.load_seqs(seq_index, seq_index + 1)
+
             tensor_dict = self._in_tensor_dict_template.copy_template()
             for data_key in data_keys:
                 tensor_dict.data[data_key].raw_tensor = self._dataset.get_data(seq_index, data_key)
+            tensor_dict.data["seq_tag"].raw_tensor = str_to_numpy_array(self._dataset.get_tag(seq_index))
```

Comment:
You can also add this to `if self._map_seq...` below, in case the user did not add it to the `tensor_dict`.

Reply:
So you would re-add it, in case `_map_seq` dropped it? I'd add the tag to the tensor dict in any case before the callback is invoked, too, in case it depends on the seq tag for e.g. FSA/alignment lookup or so.

Reply:
Yea, I think that's safe here, and a reasonable thing to do.

Reply:
Alright, done! Then this PR is good for review/merge!